notebook/static-src/edit/js/main.js

```diff
@@ -1,33 +1,22 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
+"use strict";
+
+require('base/js/globals').then(function() {
+    
+    var IPython = require('base/js/namespace');
```

Comment:
In the current notebook, the requirejs stuff was doing this anyways, now we just aren't lying to ourselves pretending that encapsulation is working.